src/modes/direct_select.js

```diff
@@ -158,6 +166,7 @@ module.exports = function(ctx, opts) {
     },
     stop: function() {
       doubleClickZoom.enable(ctx);
+      ctx.store.clearSelectedCoordinates();
```

Comment:
We need this on https://github.com/mapbox/mapbox-gl-draw/pull/533/files#diff-0210e4a9505172f8c13cd7a2cb7b6c07R164 as well.